frontend/src/components/WaiverComponent/Waiver.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Document, Page, pdfjs } from "react-pdf";
+import styled from "styled-components";
+import releaseWaiver from "./WAIVER FORM 2021.docx.pdf";
+
+pdfjs.GlobalWorkerOptions.workerSrc = `https://cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+
+const InnerWaiver = styled.div`
+  background: #c4c4c4;
+  border: 10px;
+  width: 47.81vh;
+  height: 61vh;
+`;
+
+const StyledPage = styled(Page)`
+  & .react-pdf__Page__canvas {
+    width: 47.81vh !important;
+    height: 61vh !important;
```

Comment:
Could you make the pdf twice as big? Right now the text is really hard to read. Also, you may want to look into the CSS property `overflow-y: auto;` if the pdf is too tall for the screen.